skare3_tools/config.py

```diff
@@ -104,16 +101,11 @@
 
 
 def _app_data_dir_():
-    local_app_data_dir = os.getenv("LOCALAPPDATA")
-    home_dir = os.path.expanduser("~")
+    ska_data_dir = os.path.join(os.environ["SKA"], "data", "skare3", "skare3_data")
     if "SKARE3_TOOLS_DATA" in os.environ:
         app_data_dir = os.environ["SKARE3_TOOLS_DATA"]
-    elif local_app_data_dir:
-        # this is the windows location
-        app_data_dir = os.path.join(local_app_data_dir, "skare3")
-    elif os.path.exists(home_dir) and os.access(home_dir, os.W_OK):
-        # can use this in linux and Mac OS
-        app_data_dir = os.path.join(home_dir, ".skare3")
+    elif os.path.exists(ska_data_dir):
```

Comment:
For non-head use, it looks like this will fail the first time with "Exception: Could not figure out where to place skare3_tools configuration" unless the user makes a skare3_data directory.

Reply:
That's probably fine but not sure if it needs a hint someplace in docs that I missed.

Reply:
Yes, originally I had it so it would create it in the home directory, but in reality that just needs to be set beforehand. Nothing will be created automatically.

Reply:
I changed the error message, so it tells you what to do.

Reply:
Thanks!  Though now I'm not sure if the functional test has a hiccup or the config location isn't perfect -- with or without /data?
```
Exception: Could not figure out the location of the skare3_tools configuration.
Either create the $SKA/data/skare3/skare3_data directory
or set the SKARE3_TOOLS_DATA environmental variable.

In [2]: exit
(ska3-masters) flame:skare3_tools jean$ mkdir $SKA/data/skare3/skare3_data
(ska3-masters) flame:skare3_tools jean$ ipython
Python 3.11.8 | packaged by conda-forge | (main, Feb 16 2024, 20:51:20) [Clang 16.0.6 ]
Type 'copyright', 'credits' or 'license' for more information
IPython 8.22.1 -- An enhanced Interactive Python. Type '?' for help.

In [1]: from skare3_tools import packages, config, test_results

In [2]: import os

In [3]: assert config.CONFIG['data_dir'] == f'{os.environ["SKA"]}/data/skare3/skare3_data'
---------------------------------------------------------------------------
AssertionError                            Traceback (most recent call last)
Cell In[3], line 1
----> 1 assert config.CONFIG['data_dir'] == f'{os.environ["SKA"]}/data/skare3/skare3_data'

AssertionError: 

In [4]: %debug
> <ipython-input-3-d735df94dd35>(1)<module>()
----> 1 assert config.CONFIG['data_dir'] == f'{os.environ["SKA"]}/data/skare3/skare3_data'

ipdb> config.CONFIG['data_dir']
'/Users/jean/ska/data/skare3/skare3_data/data'
```

Reply:
and what is SKA? I'm guessing there is a symlink somewhere, and I need to follow it.

Reply:
I think there's no symlink involved here on my mac.

Reply:
ok, can you show me what is `f'{os.environ["SKA"]}/data/skare3/skare3_data'`?

Reply:
Right, there seems to be this ambiguity about /data .

```
In [1]: import os

In [2]: from skare3_tools import packages, config, test_results

In [3]: f'{os.environ["SKA"]}/data/skare3/skare3_data'
Out[3]: '/Users/jean/ska/data/skare3/skare3_data'

In [4]: config.CONFIG['data_dir']
Out[4]: '/Users/jean/ska/data/skare3/skare3_data/data'
```